connectors/http/polling/src/main/java/io/camunda/connector/http/polling/model/PollingActivationProperties.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.http.polling.model;
+
+import com.fasterxml.jackson.annotation.JsonSetter;
+import com.fasterxml.jackson.annotation.Nulls;
+import io.camunda.connector.api.annotation.FEEL;
+import io.camunda.connector.generator.dsl.Property;
+import io.camunda.connector.generator.java.annotation.TemplateProperty;
+import java.time.Duration;
+
+public class PollingActivationProperties {
```

Comment:
`PollingRequest` could be a better fit as activation is usually used during correlation.

Reply:
What about this:
`PollingRequest` -> `HttpPollingConnectorProperties`
`PollingActivationProperties` -> `SchedulingProperties`
`PollingRuntimeProperties` -> `HttpPollingRequest`